examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -868,7 +868,8 @@ struct PhysicsServerCommandProcessorInternalData
 
 
 	btScalar m_physicsDeltaTime;
-    btScalar m_numSimulationSubSteps;
+	btScalar m_numSimulationSubSteps;
+	btVector3 m_gravity;
```

Comment:
This is not desired: resetSimulation should reset the state as startup, undo all things the user did (loading URDF files, setting custom settings such as gravity). It should be trivial for a user to set the gravity after doing a reset.